js/lib.globals.d.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
```

Comment:
Can you do the `git mv js/lib.deno.d.ts js/lib.globals.d.ts `  in a separate commit before this big one? (Easier to rebase and track changes)